client/js/ifml/tojson/extender.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2018, the IFMLEdit.org project authors. Please see the
+// AUTHORS file for details. All rights reserved. Use of this source code is
+// governed by a MIT-style license that can be found in the LICENSE file.
+/*jslint node: true, nomen: true */
+"use strict";
+
+var createExtender = require('almost-extend').createExtender;
+
+exports.extend = createExtender({
+    custom: {
+        isViewContainer: function (element) {
+            if (arguments.length < 1) { return false; }
```

Comment:
Do you expect to call these functions without arguments?
If not the "if guard" is not needed.